draft-ietf-masque-h3-datagram.md

```diff
@@ -161,10 +161,13 @@ HTTP/3 Datagram {
 Quarter Stream ID:
 
 : A variable-length integer that contains the value of the client-initiated
-bidirectional stream that this datagram is associated with, divided by four.
-(The division by four stems from the fact that HTTP requests are sent on
+bidirectional stream that this datagram is associated with, divided by four (The
```

Comment:
nit: if you remove the dot, then remove the following uppercase?
```suggestion
bidirectional stream that this datagram is associated with, divided by four (the
```

Reply:
derp